### ISSUE TITLE
test_single_abortable_task failed on INDEXs Pc

### ISSUE THREAD

**Issue by index-git** [jirik/layman#81]
First run failed with this message, second was OK, Travis is OK. This commit: https://github.com/jirik/layman/commit/d38d63a1a62261fbe9ac3838bbe3cc5b7b55a6ec
```python
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-6.0.1, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /code
plugins: celery-4.4.7
collected 101 items                                                                                 

src/layman/celery_test.py::test_single_abortable_task FAILED                                                              [  0%]

=========================================================== FAILURES ============================================================
__________________________________________________ test_single_abortable_task ___________________________________________________

client = <FlaskClient <Flask 'layman'>>

    def test_single_abortable_task(client):
        task_names = [
            'layman.layer.filesystem.tasks.refresh_input_chunk',
        ]
        tasks = [
            getattr(
                importlib.import_module(taskname.rsplit('.', 1)[0]),
                taskname.rsplit('.', 1)[1]
            ) for taskname in task_names
        ]
        check_crs = False
        task_options = {
            'crs_id': 'EPSG:4326',
            'description': 'bla',
            'title': 'bla',
            'ensure_user': True,
            'check_crs': check_crs,
        }
        filenames = ['abc.geojson']
        username = 'test_abort_user'
        layername = 'test_abort_layer'
        files_to_upload = input_chunk.save_layer_files_str(username, layername, filenames, check_crs)
        task_chain = chain(*[
            tasks_util._get_task_signature(username, layername, t, task_options, 'layername')
            for t in tasks
        ])
        task_result = task_chain()
    
        results = [task_result]
        results_copy = [
            AbortableAsyncResult(task_result.task_id, backend=celery_app.backend)
            for task_result in results
        ]
    
        time.sleep(1)
    
>       assert results[0].state == results_copy[0].state == 'STARTED'
E       AssertionError: assert 'PENDING' == 'STARTED'
E         - STARTED
E         + PENDING

src/layman/celery_test.py:101: AssertionError
----------------------------------------------------- Captured stdout setup -----------------------------------------------------
 * Serving Flask app "layman" (lazy loading)
 * Environment: development
 * Debug mode: off
==================================================== short test summary info ====================================================
FAILED src/layman/celery_test.py::test_single_abortable_task - AssertionError: assert 'PENDING' == 'STARTED'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================= 1 failed in 3.30s =======================================================
make: *** [Makefile:141: test] Error 1
```

**Comment by index-git** [comment]
Also on Travis CI: https://travis-ci.com/github/jirik/layman/jobs/369992507

**Comment by index-git** [comment]
Output from logging:
```results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
results[0].state=PENDING, results_copy[0].state=PENDING
```

**Comment by jirik** [comment]
OK, we need to dig more. Could you please next time also copy output of
`docker logs layman_celery_worker_test_1`
after the failure?

**Comment by jirik** [comment]
From travis:
```
$ docker logs layman_celery_worker_test_1
Waiting for Redis, url=redis://redis:6379/15
Reaching max attempts when waiting for Redis, url=redis://redis:6379/15
Waiting for Redis, url=redis://redis:6379/15
Attempt 1/60 successful.
Waiting for PostgreSQL database, {'host': 'postgresql', 'port': '5432', 'dbname': 'layman_test', 'user': 'docker'}
Reaching max attempts when waiting for PostgreSQL database, {'host': 'postgresql', 'port': '5432', 'dbname': 'layman_test', 'user': 'docker'}
```

So the reason why test_single_abortable_task failed is because celery worker was not running. And it was not running because download of data in `test.sh` took longer than usual. This is addressed in #106
